pghoard/basebackup.py

```diff
@@ -122,6 +122,14 @@ def get_command_line(self, output_name):
             "--verbose",
             "--pgdata", output_name,
         ]
+
+        # configure base backups to be standalone backups as we have disabled wal archiving
+        if self.config["backup_sites"][self.site]['active_backup_mode'] == 'standalone_hot_backup':
```

Comment:
I'm not opposed to the idea but as a minimum this needs updates to the README to describe the mode and preferably a test proving that the basebackup has the WALs included.

Reply:
fair enough, I will have a look at adding some tests and updating the read me